plutus-scb/app/Main.hs

```diff
@@ -89,6 +96,8 @@ commandParser =
              [ migrationParser
              , allServersParser
              , mockWalletParser
+             , scbWebserverParser
+             , psGeneratorCommandParser
              , mockNodeParser
              , chainIndexParser
              , signingProcessParser
```

Comment:
shouldn't line 104 just be in this list?

Reply:
Erm...sorry, I'm confused. Please elaborate... :-)

Reply:
Sorry it seems it's not possible to comment on lines that aren't in the diff. In line 104 you `<|>` together 2 `subparser` declarations, surely this is a no-op and instead of the lines below this comment you should have `contractsParser` extracted and added to this list (`command "contracts"`)

Reply:
Oh yeah! Good spot. :-)